### PR TITLE
Post game questionnaire

### PR DESCRIPTION
**Pull request by adityab17190**
This pull request contains-
1) Fix for the radio button issue on lines 274 and 275 bin trustAndTaskQuestionnaire

2) Fix for the inconsistency issue observed when navigating from post game questionnaire to thank you page.

### CODE REVIEW

File: MainApp/.gitignore [hunk 1]
```diff
@@ -8,4 +8,4 @@ tmp/
 !.gitkeep
 dist
 .idea/
-public/downloads/*
```
Comment (super9user) [inline MainApp/.gitignore:11]: Why was this removed?

File: MainApp/public/downloads/AdminLogs-1490230726013.csv [hunk 2]
```diff
@@ -0,0 +1,25 @@
+Logged At,Action,Author
```
Comment (super9user) [inline MainApp/public/downloads/AdminLogs-1490230726013.csv:1]: Pls remove.

File: MainApp/public/downloads/AdminLogs-1490229976070.csv [hunk 3]
```diff
@@ -0,0 +1,22 @@
+Logged At,Action,Author
```
Comment (super9user) [inline MainApp/public/downloads/AdminLogs-1490229976070.csv:1]: Pls remove.

File: MainApp/public/downloads/AdminLogs-1490228870888.csv [hunk 4]
```diff
@@ -0,0 +1,18 @@
+Logged At,Action,Author
```
Comment (super9user) [inline MainApp/public/downloads/AdminLogs-1490228870888.csv:1]: Pls remove.

**Review by super9user — CHANGES_REQUESTED** [review]
A few comments.

File: MainApp/.gitignore (hunk 1 above)
Comment (adityab17190) [reply MainApp/.gitignore:11]: looks like the gamedev branch is 2 commits behind the master one of which contains the 'public/downloads/*' added to gitignore. One of the commits on this branch is a merge with gamedev.

Comment (adityab17190) [reply MainApp/.gitignore:11]: @shahsyedzafar  and @tgomudur can you please look into this?

**Comment by yousuf031991** [comment]
Wait, why are gamedev commits in this PR?

File: MainApp/.gitignore (hunk 1 above)
Comment (yousuf031991) [reply MainApp/.gitignore:11]: Wait, why are gamedev commits from 10 days ago in this PR? These commits are already in master

Comment (adityab17190) [reply MainApp/.gitignore:11]: There was this bug occurring due to timeout function in gamePageCtrl.js not exiting. It was fixed by @shahsyedzafar  in this  commit https://github.com/yousuf031991/HumanAgentInteraction/pull/46/commits/7db7ceb3c386eda0000eee154522d4e61f1cf411. Thats why we see gameDev commits.

Comment (yousuf031991) [reply MainApp/.gitignore:11]: @adityab17190 Maybe you should create a new branch from master and cherry pick your own commits into it.

Comment (tgomudur) [reply MainApp/.gitignore:11]: I'm not sure why gamedev commits are showing here. But I'm removing the csvs and adding public/downloads/* to gitignore and committing.

Comment (adityab17190) [reply MainApp/.gitignore:11]: @tgomudur  and @super9user I have created new branch with following pull request #48

**Comment by adityab17190** [comment]
Duplicate

**Pull request closed without merging**